contributing.md

```diff
@@ -0,0 +1,146 @@
+
+#### Review
+
+There are two kinds of reviews:
+
+##### Functional Review
+
+In functional review, the actual behavior of the bugfix or the new feature is reviewed. If the reviewers find problems or have comments during the review, they describe the issues providing links or screenshots that show the behavior, and reassign the team member that submitted the issue, leaving the issue in the review column. If everything works as expected, the reviewers post a +1 comment on the issue and unassign themselves as reviewer.
+
+##### Code Review
+
+In code review, the technical implementation of the bugfix or the new feature is reviewed. To start a code review the person who worked on the issue assigns the corresponding pull request for code review. The pull request should be linked to the issue. Changes during the review process are created in additional commits which are pushed to the feature branch. They are added to the existing pull request automatically. At the end of the code review, the reviewer approves the pull request and reassigns the pull request to its original creator.
+
+#### Done
+
+The creator of the pull request merges the pull request after checking the [definiton of done](#definition-of-done). After the merge, the issue and the linked pull request move to the done column automatically. We delete feature branches after merging.
+
+### Definition of Ready
+
+The *Definition of Ready* describes a list of criteria which tickets have to meet to move from column 'Backlog' to 'Ready':
+
+- The person who implements the ticket is assigned and has every information to work on this ticket. Only the assignee can move the ticket to Ready.
+- The implementation of the ticket is planned for the next two (?) weeks and all participants have sufficient resources for the ticket.
+- Choose reviewers already in grooming?
+
+### Definition of Done
+
+The *Definition of Done* describes a list of criteria which tickets have to meet to be called 'Done':
+
+- CI passed (CI contains tests)
+- Code reviewed
+- Functionality reviewed (approved by user/product owner)
+
+## How can I contribute?
+
+### Reporting Bugs
```

Comment:
I think "reporting bugs" and "suggesting enhancements" should be the first paragraphs of the contributing.md to start with the easiest and lowest level of engagement by contributors.